h/static/styles/partials/_search-result-sidebar.scss

```diff
@@ -102,6 +102,22 @@
   color: $grey-6;
 }
 
+.search-result-sidebar__logo-container {
+  display: flex;
+  align-items: center;
+  margin-bottom: 26px;
```

Comment:
Totally not gonna hold up this particular PR, don't worry, but one day I'd love to see more rhythm in our proportions. 26px seems like a sort of magical number :D